examples/aidp_openai_demo/scripts/load_support_tickets.py

```
@@ -0,0 +1,376 @@
+    """Generate embeddings using NVIDIA NIM."""
+    headers = get_nvidia_headers()
+    
+    payload = {
+        "input": texts,
+        "model": EMBEDDING_MODEL,
+        "input_type": "passage",
+        "encoding_format": "float",
+        "truncate": "END"
+    }
+    
+    response = requests.post(NVIDIA_EMBED_URL, headers=headers, json=payload)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)